digitalpy/core/domain/node.py

```diff
@@ -1017,4 +1017,4 @@ def set_value(
             return result
 
         # default behaviour
-        return super().set_value(name, value, force_set, track_change)
+        return super().set_value(name, value, force_set, track_change)
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>W292 no newline at end of file